app/services/notify_app_store/http_notifier.rb

```diff
@@ -0,0 +1,15 @@
+class NotifyAppStore
+  class HttpNotifier
+    include HTTParty
+
+    def post(message)
+      self.class.post("#{host}/application/", message).parsed_response
```

Comment:
Just looking at the documentation - does this not need a content-type header? Also how well does this library handle auth?

Reply:
umm... let me check that... :) actually yes, and also check the response instead of just ignore it....